i2p_tasks.py

```diff
@@ -0,0 +1,148 @@
+from etl_tasks import SqlScriptTask
+import luigi
+from script_lib import Script
+from sql_syntax import Environment
+from typing import List
+
+class Condition(SqlScriptTask):
+    script = Script.condition
+
+    def requires(self) -> List[luigi.Task]:
+        return [Encounter()]
+
+class Death(SqlScriptTask):
+    script = Script.death
+
+    def requires(self) -> List[luigi.Task]:
+        return [Demographic()]
+
+class DeathCause(SqlScriptTask):
+    script = Script.death_cause
+
+class Demographic(SqlScriptTask):
+    script = Script.demographic
+
+    def requires(self) -> List[luigi.Task]:
+        return [PcornetInit()]
+
+class Diagnosis(SqlScriptTask):
+    script = Script.diagnosis
+
+    def requires(self) -> List[luigi.Task]:
+        return [Encounter()]
+
+    @property
+    def variables(self) -> Environment:
+        return dict(i2b2_meta_schema='BLUEHERONMETADATA')
+
+class Dispensing(SqlScriptTask):
+    script = Script.dispensing
+
+    def requires(self) -> List[luigi.Task]:
+        return [Encounter()]
+
+    @property
+    def variables(self) -> Environment:
+        return dict(i2b2_meta_schema='BLUEHERONMETADATA')
+
+
+class Encounter(SqlScriptTask):
+    script = Script.encounter
+
+    def requires(self) -> List[luigi.Task]:
+        return [Demographic()]
+
+class Enrollment(SqlScriptTask):
+    script = Script.enrollment
+
+    @property
+    def variables(self) -> Environment:
+        return dict(enrollment_months_back='2')
+
+class Harvest(SqlScriptTask):
+    script = Script.harvest
+
+    def requires(self) -> List[luigi.Task]:
+        return [Condition(), Death(), Diagnosis(), Dispensing(), LabResultCM(), Prescribing(), Procedures(), Vital()]
+
+    @property
+    def variables(self) -> Environment:
+        return dict(network_id='C4', network_name='GPC')
+
+class LabResultCM(SqlScriptTask):
+    script = Script.lab_result_cm
+
+    def requires(self) -> List[luigi.Task]:
+        return [Encounter()]
+
+class MedAdmin(SqlScriptTask):
```

Comment:
i2p_tasks.py is the Luigi interface module.  In the initial release, Luigi will only be calling the MedAdmin function.  MedAdmin relies on an init task the checks the observation_fact table for useful data.

Otherwise, the remaining code in this file and supporting SQL files is inactive.